app-modules/engagement/src/Filament/Actions/RelationManagerSendEngagementAction.php

```diff
@@ -327,16 +340,7 @@ protected function setUp(): void
             ->modalCloseButton(false)
             ->closeModalByClickingAway(false)
             ->closeModalByEscaping(false)
-            ->createAnother(false)
-            ->modalCancelAction(false)
-            ->extraModalFooterActions([
```

Comment:
I think this custom "cancel" action needs to stay somehow, otherwise is the TipTap editor bug still there when you close and reopen the modal?